js/common.js

```diff
@@ -1658,6 +1658,7 @@ var App = new function() {
 
                 act.onsuccess = function() {
                     if (!act.result.blob) return;
+                    elSave.classList.add('visible');
```

Comment:
this might get out of sync if the `CLASS_WHEN_VISIBLE` variable is changed inside the `NoteView`... I would probably try to expose and reuse the same logic as `onContentKeyUp` and rename/expose it to something like `toggleDoneButton` and/or maybe even expose 2 separate methods `showDoneButton` and `hideDoneButton`.

just thinking that if we keep adding more logic like this everywhere it will be harder to update things without introducing regressions.